pyproject.toml

```diff
@@ -27,12 +28,11 @@ description = "OpenAlea.HydroRoot is a hydraulic root architecture modelling and
 readme = "README.rst"
 license = "CECILL-C"
 license-files = ["LICEN[CS]E*"]
-requires-python = ">=3.6"
+requires-python = ">=3.9"
```

Comment:
I will not restrict Python version here.
I will restrict in the github CI conf file the versions of Python